### PR TITLE
Develop to Main

### PR DESCRIPTION
**Pull request by tbddorothy**

### CODE REVIEW

File: RR.App/token.json/Google.Apis.Auth.OAuth2.Responses.TokenResponse-user
Comment (PleaBanshee) [inline RR.App/token.json/Google.Apis.Auth.OAuth2.Responses.TokenResponse-user:1]: Just to confirm, should these be pushed to main?

**Review by AWallsRR — APPROVED** [review]
Only leaving the one comment about a comment i found but I'm approving assuming we don't care about the comment for this and it will be removed later

File: ATS.Models/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace ATS.Models
```
[the remark this answers is not in the document]
Comment (tbddorothy) [reply ATS.Models/Class1.cs:1]: Just something to have in the Models folder

**Review by PleaBanshee — APPROVED** [review]

File: HRIS.Models/EmployeeAddressDto.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿namespace HRIS.Models;
+
+public class EmployeeAddressDto
+{
+    // TODO : Remove this Constructor + Update UNIT TESTS
```
[the remark this answers is not in the document]
Comment (tbddorothy) [reply HRIS.Models/EmployeeAddressDto.cs:5]: We're just updating the main branch to be in the same state as develop. All these changes will be made

File: HRIS.Models/EmployeeDto.cs [hunk 3]
```diff
@@ -1,48 +1,48 @@
-﻿using RGO.Models.Enums;
+﻿using HRIS.Models.Enums;
 
-namespace RGO.Models;
+namespace HRIS.Models;
 
 public class EmployeeDto
 {
     // TODO : Remove this Constructor + Update UNIT TESTS
     public EmployeeDto(int Id,
```
[the remark this answers is not in the document]
Comment (tbddorothy) [reply HRIS.Models/EmployeeDto.cs:8]: We're just getting main to the same state as develop for the rewrite. This work is in progress

File: RR.App/token.json/Google.Apis.Auth.OAuth2.Responses.TokenResponse-user
Comment (tbddorothy) [reply RR.App/token.json/Google.Apis.Auth.OAuth2.Responses.TokenResponse-user:1]: Yeah this is a generated token from long ago. Will have it removed from develop

**Review by ZuenTheMan — APPROVED** [review]

**Review by LlewellynAnthony — APPROVED** [review]

**Review by AWallsRR — APPROVED** [review]

**Pull request merged**